code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -491,3 +491,56 @@
 		if(L.heal_damage(heal_amt/parts.len, heal_amt/parts.len, null, BODYPART_ORGANIC))
 			M.update_damage_overlays()
 	return 1
+
+/datum/symptom/heal/surface
+	name = "Superficial Healing"
+	desc = "The virus accelerates the body's natural healing, causing the body to heal minor wounds quickly. Causes heavy scarring."
+	stealth = -1
+	resistance = -2
+	stage_speed = -2
+	transmittable = 1
+	level = 6
+	passive_message = "<span class='notice'>Your skin tingles.</span>"
+	var/threshhold = 15
+	var/scarcounter = 0
+	threshold_descs = list(
+		"Stage Speed 8" = "Doubles healing speed.",
+		"Resistance 10" = "Improves healing threshhold.",
+	)
+
+/datum/symptom/heal/surface/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["stage_rate"] >= 8) //stronger healing
+		power = 2
+	if(A.properties["resistance"] >= 10)
+		threshhold = 30
+
+/datum/symptom/heal/surface/Heal(mob/living/carbon/M, datum/disease/advance/A, actual_power)
+	var/healed = FALSE
+
+	if(M.getBruteLoss() && M.getBruteLoss() <= threshhold)
+		M.adjustBruteLoss(-power)
+		healed = TRUE
+		scarcounter++
+
+	if(M.getFireLoss() && M.getFireLoss() <= threshhold)
+		M.adjustFireLoss(-power)
+		healed = TRUE
+		scarcounter++
+
+	if(M.getToxLoss() && M.getToxLoss() <= threshhold)
+		M.adjustToxLoss(-power)
+		healed = TRUE
+
+	if(healed)
+		if(prob(10))
+			to_chat(M, "<span class='notice'>Your wounds heal, granting you a new scar</span>")
```

Comment:
this doesn't check if the scars actually increase, and can trigger off the toxin healing which does't grant one